README.md

```diff
@@ -1,6 +1,69 @@
 # curriculum-cr-training-mobile-menu
 
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Portfolio Website
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+> This is a portfoli project to showcse my coding projects and contact information on the web.
+
+## Built With
+
+- Html
+- Css
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://ginohmk.github.io/Ham_menu_code_review-training/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- node
+- web broswer
+- code editor
+
+### Setup
+
+```
+  git clone git@github.com:Ginohmk/Ham_menu_code_review-training.git
+```
+
+```
+ cd Ham_menu_code_review-training
+```
+
+### Usage
+
+` open index.html in web broswer`
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] kindly you should change this template details to your social media links to make your project look professional and reflect you as the author of the project. doing this will also help people who will want to know more about the author of the project.You got this!💪🏾